language/fr/logout.lang

```diff
@@ -0,0 +1,8 @@
+<?php
+
+include("{$_EnginePath}language/en/logout.lang");
```

Comment:
This file's translation is completed, therefore you can remove this default import.

Reply:
Sorry, I may have misunderstood your comment below:
> BTW, this solution should also work just fine even if new translation strings will be added only partially to a file - leaving the include as the first line, and then adding new strings will simply override the English translation strings in runtime.
I though you suggested we included this include line in every single translation file other than English to have a fallback language file if new entries were added in the English language files in the future. Could you confirm you would like these include instructions to be used ONLY when a translation file (other than English) is not complete?

Reply:
> Could you confirm you would like these include instructions to be used ONLY when a translation file (other than English) is not complete?

Yes, that's exactly what I meant in my previous comments. Sorry if I didn't make this more clear.

Ultimately, I would like to modify the translation system itself to prevent empty messages from being displayed by defaulting to English automatically under the hood. This will most likely require a major refactor of said system, but I already have an idea or two how this process could be made incremental with some temporary workarounds. That's why I would like that all completed translations are already without these fallback defaults.

Reply:
Perfect. I understand. I will apply that.

Reply:
Done.